create-dev-env.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-
 # Checks if a binary is present on the local system
+export POSTGRESQL_DATABASE="postgres"
```

Comment:
If the `POSTGRESQL_DATABASE` environment variable is already set by the user calling the script, we should use that value, otherwise, if it's blank, then we should default to `postgres`. See [here](https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash) for how that looks in bash script.